addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -22,8 +22,8 @@ var LIVECHAT_COOKIE_HISTORY = 'im_livechat_history';
 var HISTORY_LIMIT = 15;
 
 var RATING_TO_EMOJI = {
-    "10": "😊",
-    "5": "😐",
+    "5": "😊",
```

Comment:
good call, 10 was way too much happiness ! Serious question : doesn't this have an impact on existing ratings and shouldn't this be included in master (with migration)?

Reply:
actually, new values of rating are 5, 3 and 1. I just aligned this with rating values.

Reply:
Duplicate open PR https://github.com/odoo/odoo/pull/64518 where I linked the commit where the change to number was done for reference.

Reply:
Well, au pire, there is more rating broken stuffs in that commit.

Reply:
Could be great to link original PR doing the change.